client/jetpack-cloud/sections/partner-portal/primary/payment-method-add/index.tsx

```diff
@@ -73,7 +78,28 @@ function PaymentMethodAdd() {
 		[ window.location.href, getQueryArg ]
 	);
 
+	const products = useMemo(
+		() => ( getQueryArg( window.location.href, 'products' ) || '' ).toString(),
+		[]
+	);
+
+	const siteId = useMemo(
+		() => getQueryArg( window.location.href, 'site_id' ) || '',
+		[]
+	).toString();
```

Comment:
Can we change this to something like this (see `products` above) for consistency?

```
	const siteId = useMemo(
		() => ( getQueryArg( window.location.href, 'site_id' ) || '').toString(),
		[]
	);
```